src/fides/api/ops/service/connectors/consent_email_connector.py

```diff
@@ -0,0 +1,170 @@
+from typing import Any, Dict, List, Optional
+
+from loguru import logger
+from sqlalchemy.orm import Query, Session
+
+from fides.api.ctl.sql_models import Organization  # type: ignore[attr-defined]
+from fides.api.ops.common_exceptions import MessageDispatchException
+from fides.api.ops.models.connectionconfig import (
+    AccessLevel,
+    ConnectionConfig,
+    ConnectionTestStatus,
+    ConnectionType,
+)
+from fides.api.ops.schemas.connection_configuration.connection_secrets_email_consent import (
+    AdvancedSettingsWithExtendedIdentityTypes,
+    ExtendedConsentEmailSchema,
+    ExtendedIdentityTypes,
+)
+from fides.api.ops.schemas.connection_configuration.connection_secrets_sovrn import (
+    SOVRN_REQUIRED_IDENTITY,
+)
+from fides.api.ops.schemas.messaging.messaging import (
+    ConsentEmailFulfillmentBodyParams,
+    ConsentPreferencesByUser,
+    MessagingActionType,
+)
+from fides.api.ops.schemas.privacy_request import Consent
+from fides.api.ops.schemas.redis_cache import Identity
+from fides.api.ops.service.connectors.base_connector import LimitedConnector
+from fides.api.ops.service.messaging.message_dispatch_service import dispatch_message
+from fides.core.config import get_config
+
+CONFIG = get_config()
+
+CONSENT_EMAIL_CONNECTOR_TYPES = [ConnectionType.sovrn]
+
+
+class GenericEmailConsentConnector(LimitedConnector[None]):
+    """Generic Email Consent Connector that can be overridden for specific vendors"""
+
+    @property
+    def identities_for_test_email(self) -> Dict[str, Any]:
+        """The mock user identities that are sent in the test
+        email to ensure the connector is working"""
+        return {"email": "test_email@example.com"}
```

Comment:
just so I understand, this email doesn't actually show up anywhere, right? in the test email I got, I have the cookie name `test_ljt_reader_id` but I don't see `test_email@example.com` anywhere.